src/js/Controller/Setting/Get.js

```diff
@@ -29,7 +29,8 @@ export default class Get extends AbstractController {
             'password.list.show.user',
             'mining.ignored-domains',
             'mining.incognito.hide',
-            'autofill.ignored-domains'
+            'autofill.ignored-domains',
+            'contextmenu.disable'
```

Comment:
Can you use a positive label: `contextmenu.enabled`
I'd like to keep this in line with the other settings